src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.Gui.Gui;
+import duke.component.Storage;
+import duke.task.TaskList;
+
+
+import java.util.ArrayList;
+
+public class ByeCommand extends Command {
+
+    /**
+     * Used to show bye message to user
```

Comment:
Perhaps reconsider using documentation to explain purpose of code. Maybe can include after first explaining what the code does.